users/consumers.py

```
@@ -24,118 +24,52 @@ def handle_message(self, body: str, routing_key=None):
+                        self.logger.info(
+                            f"User {user_id} deleted successfully",
+                            extra={"user_id": user_id, "event": "user.deleted"},
+                        )
+                    else:
+                        self.logger.warning(
+                            f"User {user_id} not found for deletion",
+                            extra={"user_id": user_id, "event": "user.deleted"},
+                        )
+             
+                case _:
+                        raise Exception("Failed to process user event due to incorrect event type")
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)